src/main/java/seedu/eduke8/Parser.java

```diff
@@ -1,13 +1,27 @@
 package seedu.eduke8;
```

Comment:
I think it is better if the parser class and its interface could be packaged into their own package, which can be called parser